log4j-api/src/test/java/org/apache/logging/log4j/util/ConstantsTest.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.logging.log4j.util;
+
+import org.junit.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Properties;
+
+import static org.junit.Assert.*;
+
+public class ConstantsTest {
+
+    @Test
+    public void testJdkVersionDetection() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
```

Comment:
nit: No need to declare these exception types now that we're calling `getMajorVersion` directly.

Reply:
The code has been cleaned up.